torch/testing/_internal/common_utils.py

```diff
@@ -2233,6 +2233,22 @@ def setUp(self):
         check_if_enable(self)
         set_rng_seed(SEED)
 
+        # Save global check sparse tensor invariants state that can be
+        # restored from dearDown:
+        self._check_invariants = torch.is_check_sparse_tensor_invariants_enabled()
+
+        # Enable invariant checks for all sparse tensors constructions
+        # including the unsafe ones. If this is not desired for some
+        # test case, use check_invariants=False optional argument to
+        # sparse tensor constructors or
+        # @CheckSparseTensorInvariants(False) decorator from
+        # test/test_sparse.py to disable the invariant checks.
+        torch.enable_check_sparse_tensor_invariants(True)
```

Comment:
Technically you could pass `False`. Wouldn't `set_[...]` make more sense than `enable_[...]`?

Reply:
Yes, it makes sense.

`torch.enable_check_sparse_tensor_invariants(<enabled>)` is now replaced with a static method pair:
```
torch.sparse.check_sparse_tensor_invariants.enable()
torch.sparse.check_sparse_tensor_invariants.disable()
```